### DISCUSSION TITLE
셀카모드처럼 찍은 사진을 스마트폰 로컬에 저장하는 방법

### DISCUSSION THREAD

**Discussion by xowjd913** [naverz/zepeto-studio-kor#108] [5. Q&A]
현재 다른 월드 처럼 셀카를 찍어 스마트폰에 저장할 수 있는 방법이 있을까요 ?
File IO 쪽은 엄청 보수적일것같은데 ...

**Comment by hyeonjin-jo** [comment]
안녕하세요.

현재는 해당 기능을 제공하지 않고 있습니다.

감사합니다.

**Reply by xowjd913** [reply]
답변 감사합니다.
혹시 그럼 촬영한 사진을 피드하거나 공유 할 수 있는 기능은 따로 제공하고있나요 ?

**Reply by hyeonjin-jo** [reply]
추후 논의될 수는 있을 것으로 보이나 현재는 제공하지 않고 있습니다.

**Reply by xowjd913** [reply]
그럼혹시 다른월드에는 제공이 되고있던데
타플랫폼 sns 공유나 피드기능은 제페토와 제휴를 맺어야
제공을 해주는건가요?

**Reply by cheolhui-lee** [reply]
기존 월드는 스튜디오를 통해 제작된 것이 아니라 내부에서 별도로 직접 만든 맵들입니다.
현재 제페토 스튜디오에서는 해당 기능이 아직 제공되고 있지 않습니다만,
우선순위에 따라서 기존 월드의 기능 중에 일부를 제공할 수 있도록 준비하고 있는 상황입니다.

**Reply by xowjd913** [reply]
답변 감사합니다 :) 점심은 맛있게 드셨습니까.
그럼 회사측과 제휴를 맺더라도 해당 기능은 제공해주는데 시간이 걸릴수 있다고 이해하면 될까요?

**Reply by cheolhui-lee** [reply]
넵. 제휴와 무관하게 기능 개발이 우선이고 기능 개발 작업이 언제 시작될 지 아직 정해진 부분이 없으므로 현재는 불가능 하다고 보시는 것이 좋겠습니다.

**Reply by xowjd913** [reply]
답변 감사합니다.
재차 질문 드려서 죄송합니다. 이번 질문이 마지막 질문이 될것같습니다 :)

1. **현재 Zepeto World 탬플릿을 통해 기능을 커스터마이징해 개발을 진행 하려합니다.**
    **별도의 다른 아바타와 Zepeto 캐릭터를 포토존에 세워놓고 사진을 찍은뒤 해당 사진을 폰에 저장하는 기능을 구현하려합니다.
    현재도 월드에 Zepeto Korea 이름을 갖고 있지 않은 일반 제작자(?) 들도 월드에 셀카모드가 구현되어있는데 
    아예 해당 기능을 구현하거나 오픈해주시는게 불가능하다는 말씀으로 이해하면 될까요?**
    혹시 빌드잇으로 월드를 서비스 하면 해당 내용이 추가되어 들어가는지도 궁금합니다.

2. 현재 다른 월드를 들어가보면 아이탬 구매 기능이 구현되어있는데 해당 기능을 구현하지 않아도 월드를 배포하는데
    크게 문제가 없을까요 ? 

3. Zepeto 캐릭터에 추가로 애니메이션 클립을 만들어 사용하려 합니다.
    해당 애니메이션 클립을 사용하기 위한 UI는 별도로 개발해도 월드를 배포하는데 크게 문제가 없을까요 ?
